File: Assigment 3/Python/06. inter change diff row of a matrix .py

arr = [[1, 1, 1, 1, 1],
       [2, 2, 2, 2, 2],
       [3, 3, 3, 3, 3],
       [4, 4, 4, 4, 4],
       [5, 5, 5, 5, 5]]

m = int(input("enter row 1: "))
n = int(input("rnter row 2: "))

m-=1
n-=1

for i in range(5):
    t = arr[n][i]
    arr[n][i] = arr[m][i]
    arr[m][i] = t

print(arr)

File: Assigment 3/C/06. inter change diff row of a matrix .c
#include <stdio.h>
int main(){
    int arr[][5] = {{1,1,1,1,1},
                    {2,2,2,2,2},
                    {3,3,3,3,3},
                    {4,4,4,4,4},
                    {5,5,5,5,5}};

    int a,b;
    printf("enter a & b: \n");
    scanf("%d%d",&a,&b);

    for(int i=0; i<5; i++){
        int t = arr[a-1][i];
        arr[a-1][i] = arr[b-1][i];
        arr[b-1][i] = t;
    }

    //print
    for(int i=0; i<5; i++){
        for(int j=0; j<5; j++){
            printf("%d ",arr[i][j]);
        }
        printf("\n");
    }
    return 0;
}
